scatter-gather/scatter-gather.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/http;
+
+type CarRental record {|
+    string carType;
+    string price;
+    string resultLink;
+|};
+
+service /api/v1 on new http:Listener(8080) {
+    
+    resource function get vehicles(string dropOffDateTime, string dropOffLocation, string pickupDateTime, string pickupLocation) returns CarRental[]|error {
+        return requestRentalCar(dropOffDateTime, dropOffLocation, pickupDateTime, pickupLocation);
+    }
+}
+
+function requestRentalCar(string dropOffDateTime, string dropOffLocation, string pickupDateTime, string pickupLocation) returns CarRental[]|error {
```

Comment:
inline this function.

Reply:
Do you mean to have this body inside the resource function?

Reply:
yes.